packages/data-stores/src/starter-designs-queries/constants.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Standard themes that are verticalizable
+ */
+export const STANDARD_THEMES_V13N_WHITELIST = [
+	'pub/geologist',
```

Comment:
Just realized, should we also include Meraki, Heiwa, Appleton? As they are somehow also included in the static designs 😂

Reply:
It depends. If we think the result looks good, we could. But it might be a little duplicated with the generated designs

Reply:
Cool, let's not do it for now.